cmd/cloud-controller-manager/.import-restrictions

```diff
@@ -9,6 +9,7 @@ rules:
       - k8s.io/kubernetes/cmd/cloud-controller-manager/app/options
       - k8s.io/kubernetes/cmd/controller-manager/app
       - k8s.io/kubernetes/cmd/controller-manager/app/options
+      - k8s.io/kubernetes/cmd/controller-manager/pkg/features
```

Comment:
Should this be `k8s.io/controller-manager/pkg/features`?

Reply:
Or removed I guess since this list is only for `k8s.io/kubernetes` imports.

Reply:
Updated to remove this import ^^